src/e2e-test/java/io/cdap/cdap/ui/utils/Helper.java

```diff
@@ -87,4 +102,74 @@ public static String getNodeSelectorFromNodeIndentifier (NodeInfo node) {
       node.getNodeType() + "-" +
       node.getNodeId() + "\"]";
   }
+
+  public static void deployAndTestPipeline(String filename, String pipelineName) {
+    SeleniumDriver.openPage(Constants.PIPELINES_URL);
+    WaitHelper.waitForPageToLoad();
+    ElementHelper.clickOnElement(locateElementById("resource-center-btn"));
+    ElementHelper.clickOnElement(locateElementById("create-pipeline-link"));
+    if(!SeleniumDriver.getDriver().getCurrentUrl().contains("studio")) {
+      throw new RuntimeException("URL redirection for pipeline creation unsuccessful.");
+    }
+    WaitHelper.waitForPageToLoad();
+    WebElement upload_file = SeleniumDriver.getDriver()
+            .findElement(By.xpath("//*[@id='pipeline-import-config-link']/input[@type='file']"));
+
+    File pipelineJSONFile = new File("src/e2e-test/fixtures/" + filename);
```

Comment:
you can create a constant for `fixtures dir` in `Constants` class like `FIXTURES_DIR="src/e2e-test/fixtures/"`

Reply:
yes, done